src/shared/libfido2-util.c

```diff
@@ -1214,7 +1224,7 @@ int fido2_find_device_auto(char **ret) {
                 goto finish;
         }
 
-        r = check_device_is_fido2_with_hmac_secret(sym_fido_dev_info_path(entry));
+        r = check_device_is_fido2_with_hmac_secret(sym_fido_dev_info_path(entry), NULL, NULL, NULL, NULL);
```

Comment:
our current coding style says when calling functions which parameters that are too opaque and give no hint on what they are, to prefix them with a comment with their name. i.e. change the first `NULL` here to `/* ret_has_rk= */ NULL` and so on.

(and then maybe line break the parameters, since this would get too long

Reply:
oops, sorry